### ISSUE TITLE
etc

### ISSUE THREAD

**Issue by robi09** [Codeinwp/Denta-Lite#4]
Acest rep a fost facut cu numele gresti, il pot sterge? @ineagu

**Comment by ineagu** [comment]
Pai nu schimbasem tema cu numele asta?

**Comment by robi09** [comment]
@ineagu  nope, dentatheme-lite a fost cel schimbat. aici nu s-a lucrat deloc.
